build-logic/src/main/kotlin/com.hedera.hashgraph.benchmark-conventions.gradle.kts

```diff
@@ -17,10 +17,9 @@ plugins {
     id("me.champeau.jmh")
 }
 
-val includesRegex: String by project
 jmh {
     jmhVersion.set("1.36")
-    includes.set(listOf(includesRegex))
+    includes.add(".*")
```

Comment:
Let's avoid having custom properties in 'gradle.properties' files. All custom configuration of a project is best put into its build.gradle.kts file to have everything in one place. Here however, the value was always `.*` anyway. The `gradle.properties` files with `includesRegex` are now removed.

Reply:
@jjohannes This was a properties file entry to allow users to limit which JMH benchmarks were run and to provide a default value if not otherwise specified on the command line. 

@OlegMazurov do you use this property?

Reply:
I had to set `includesRegex=.*` in `gradle.properties` for a reason unknown to me. I guess I can say I do not use it.

Reply:
I removed this now, as including everything is the default.

If giving a regex in local development as parameter is useful, I can reintroduce this as an optional command line parameter. So that you can call

`./gradlew :module-name:jmh -PincludesRegex='smth.*'`

to only run parts of the benchmarks locally.

But if it's not used, I would prefer to not add it now. We can always introduce that again later.